app/src/main/java/com/github/dedis/student20_pop/model/Lao.java

```diff
@@ -133,6 +133,8 @@ public String getAttestation() {
     public Lao setName(String name) {
         if (name == null) {
             throw new IllegalArgumentException("Trying to set null as the name of the LAO");
+        } else if (name.isEmpty()) {
```

Comment:
Maybe we should use isBlank(), to avoid having a name with only whitespaces

Reply:
I can't find any isBlank() declaration, but what we could do is call trim() on the name to remove leading and trailling whitespace and then checking for emptiness

Reply:
perfect

Reply:
done